src/test/debuginfo/enum-thinlto.rs

```diff
@@ -0,0 +1,48 @@
+// ignore-tidy-linelength
+
+// Require LLVM with DW_TAG_variant_part and a gdb that can read it.
+// min-system-llvm-version: 7.0
```

Comment:
To keep this in sync with #59377, this should check for 8.0.

Reply:
Seconded.